packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/EditContract.jsx

```diff
@@ -178,6 +185,21 @@ const EditContract = props => {
               disabled
               variant={fieldVariant}
             />
+            {policy.setSync && flag('harvest.toggle-contract-sync') ? (
+              <FieldContainer variant={fieldVariant}>
+                <Label for="account-sync-switch">
```

Comment:
Are we using this `account-sync-switch` id somewhere? I didn't find it

Reply:
So no need for it, we can remove it, right? Or we can add this ID to the Switch

Reply:
Yes I've removed it.